assets/styles/_responsive.scss

```diff
@@ -132,7 +143,7 @@
   }
 
   .header-carousel {
-    height: 99vh !important;
+    height: 110vh !important;
```

Comment:
let's refrain from changing heights and widths from now. let's focus on the fonts and typography.